osmosis-1/osmosis.zone_assets.json

```diff
@@ -5719,6 +5719,16 @@
         "meme"
       ],
       "_comment": "Sherpa $SHERPA"
+    },
+    {
+      "chain_name": "neutron",
+      "base_denom": "factory/neutron1shwxlkpdjd8h5wdtrykypwd2v62z5glr95yp0etdcspkkjwm5meq82ndxs/amatom",
+      "path": "transfer/channel-874/factory/neutron1shwxlkpdjd8h5wdtrykypwd2v62z5glr95yp0etdcspkkjwm5meq82ndxs/amatom",
+      "osmosis_verified": false,
+      "categories": [
```

Comment:
This is not a valid category--please remove

Reply:
changed to "defi"